content/pt-br/docs/tasks/access-application-cluster/access-cluster-services.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Acessando serviços em execução em clusters
+content_type: task
+weight: 140
+---
+
+<!-- overview -->
+Esta página mostra como se conectar aos serviços em execução no cluster Kubernetes.
+
+{{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Acessando serviços em execução no cluster
+
+No Kubernetes, [nós](/pt-br/docs/concepts/architecture/nodes/), [pods](/docs/concepts/workloads/pods/) e [serviços](/docs/concepts/services-networking/service/) têm todos seus próprios IPs. Em muitos casos, os IPs dos nós, dos pods e alguns dos IPs de serviço em um cluster não serão
+roteáveis, portanto, não estarão acessíveis a partir de uma máquina fora do cluster, como seu computador de mesa.
```

Comment:
"Computador de mesa" sounds a little weird to me. Perhaps one of "computador pessoal", "estação de trabalho", or even just "computador" could fit better here?